libantworld/common.h

```diff
@@ -1,6 +1,30 @@
 #pragma once
 
+// Third-party includes
+#include "../third_party/units.h"
+
 //----------------------------------------------------------------------------
 // Macros
 //----------------------------------------------------------------------------
-#define BUFFER_OFFSET(i) ((void*)(i))
\ No newline at end of file
+#define BUFFER_OFFSET(i) ((void*)(i))
+
+namespace BoBRobotics {
+namespace AntWorld {
+inline constexpr units::length::meter_t makeM(const GLfloat value)
+{
+    return units::make_unit<units::length::meter_t>(value);
+}
+
+template<class T1, class T2>
+auto distance2(const T1 &v1, const T2 &v2)
+{
+    return hypot(v2[1] - v1[1], v2[0] - v1[0]);
+}
+
+template<class T>
+units::length::meter_t distance2(const T &v1, units::length::meter_t x2, units::length::meter_t y2)
```

Comment:
I just remembered I meant to rename these to ``distance()`` because they aren't distance squared any more...

Reply:
I figured that the removal of square roots at all costs optimisation was probably not really necessary these days....